PackageExplorer/PublishPackageWindow.xaml

```diff
@@ -59,8 +59,8 @@
 
         <!-- Publish key -->
         <DockPanel x:Name="PanelApiKey" DockPanel.Dock="Top" Margin="7,0,10,5">
-            <Label DockPanel.Dock="Left" Target="{Binding ElementName=PublishKey}" Content="Publish _key:" Width="75" />
-            <TextBox x:Name="PublishKey" VerticalContentAlignment="Center" IsEnabled="{Binding CanPublish}" Margin="0" Text="{Binding PublishKey}">
+            <Label DockPanel.Dock="Left" Target="{Binding ElementName=PublishKey}" Content="Publish _key or PAT:" Width="110" />
```

Comment:
Using Personal Access Token here does seem more clear

Reply:
The problem is that it is quite long:
![image](https://user-images.githubusercontent.com/4009570/36357252-c4a91550-14fb-11e8-8b30-b22785a16d4d.png)

What about this?
![image](https://user-images.githubusercontent.com/4009570/36357261-dfd7acce-14fb-11e8-8445-d31e15a8ae6b.png)

Reply:
that looks good, just the first time.

Reply:
Done.